pynicotine/slskmessages.py

```diff
@@ -301,21 +313,21 @@ def getObject(self, message, type, start=0, getintasshort=0, getsignedint=0, pri
         a binary array). start is an offset."""
         intsize = struct.calcsize("<I")
         try:
-            if type is types.IntType:
+            if type is int:
                 if getintasshort:
                     return intsize+start, struct.unpack("<H", message[start:start+struct.calcsize("<H")])[0]
                 elif getsignedint:
                     return intsize+start, struct.unpack("<i", message[start:start+intsize])[0]
                 else:
                     return intsize+start, struct.unpack("<I", message[start:start+intsize])[0]
-            elif type is types.LongType:
+            elif type is int:
```

Comment:
Well, dang. I'm not sure how to handle this as I have no idea what the protocol expects - I don't think the SLSK protocol is documented anywhere. So it is really hard for me to determine the original intention here.
However, since I only find calls to `getObject()` which request to get a long int made with `NetworkLongLongType` as `type` (I'll need to get rid of variables which are named the same as builtins btw, but I'll defer that to a different PR since this one is big and comlex enough already) I'm going to delete this block. `NetworkLongLongType` is handled below.